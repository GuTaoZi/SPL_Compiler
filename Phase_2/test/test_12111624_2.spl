int ss(int a, int b){
    return ss(a,b+a);
}
int main(){

}
