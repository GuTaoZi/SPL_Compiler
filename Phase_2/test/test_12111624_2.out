Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: ss
        LP
        VarList (1)
          ParamDec (1)
            Specifier (1)
              TYPE: int
            VarDec (1)
              ID: a
          COMMA
          VarList (1)
            ParamDec (1)
              Specifier (1)
                TYPE: int
              VarDec (1)
                ID: b
        RP
      CompSt (1)
        LC
        StmtList (2)
          Stmt (2)
            RETURN
            Exp (2)
              ID: ss
              LP
              Args (2)
                Exp (2)
                  ID: a
                COMMA
                Args (2)
                  Exp (2)
                    Exp (2)
                      ID: b
                    PLUS
                    Exp (2)
                      ID: a
              RP
            SEMI
        RC
    ExtDefList (4)
      ExtDef (4)
        Specifier (4)
          TYPE: int
        FunDec (4)
          ID: main
          LP
          RP
        CompSt (4)
          LC
          RC
